packages/grant-explorer/src/features/common/PassportWidget.tsx

```diff
@@ -0,0 +1,137 @@
+import { useState } from "react";
+import { PassportState, usePassport } from "../api/passport";
+import { useAccount } from "wagmi";
+import { ReactComponent as GitcoinPassportLogo } from "../../assets/passport-logo.svg";
+import { ReactComponent as GitcoinPassportBWLogo } from "../../assets/passport-logo-bw.svg";
+import { ReactComponent as GitcoinPassportLogoFull } from "../../assets/passport-logo-full.svg";
+import { Dropdown as DropdownIcon } from "common/src/icons/Dropdown";
+
+export function PassportWidget() {
+  const { address } = useAccount();
+
+  const { passportState, passportScore, passportColor, donationImpact } =
+    usePassport({ address });
+
+  const [open, setOpen] = useState(false);
```

Comment:
`isXyz` would be a clearer name for boolean parameters than `xyz` (can't get mistaken for an action)

Reply:
Done.